webrtc-stats.html

```diff
@@ -1920,10 +1921,11 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  Exposes the {{RTCRtpCodingParameters/rid}}
-                  encoding parameter of this <a>RTP stream</a> if it has been set, otherwise it is
-                  undefined. If set this value will be present regardless if the RID RTP header
-                  extension has been negotiated.
+                  Does not [= map/exist =] for audio.
```

Comment:
nit: Can you remove the newline here and adjust this paragraph to not exceed the per-line char limit?